js/hitbtc.js

```diff
@@ -805,6 +804,37 @@ module.exports = class hitbtc extends Exchange {
         let response = await this.tradingGetOrdersRecent (this.extend (request, params));
         return this.parseOrders (response['orders'], market, since, limit);
     }
+    
+    async fetchOrderTrades (id, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let response = await this.tradingGetTradesByOrder (this.extend ({
+            'clientOrderId': id,
+        }, params));
+        return this.parseOrderTrades (response['trades'], since, limit);
+    }
+    
+    parseOrderTrades (trades, since = undefined, limit = undefined) {
+        let result = Object.values (trades).map (trade => this.parseOrderTrade (trade));
```

Comment:
This line violates the rules [from here](https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes), this one in particular:
> - unfold all maps and comprehensions to basic for-loops

We can't really map functions in cross-language-portable way like you're used to do when you code in JS-only environment. Therefore the for loops. Btw, `for` is 10x faster )

Can you unfold this one, please?

Reply:
Will unfold it , I just use parseOrder from base/Exchange.js , forgot base were language specifics

Reply:
Yeah, it takes time to get used to it, but we are restricted from some of convenient instruments, for portability reasons.

Reply:
Maybe possible to add that check to eslint no ?